.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,21 +1,18 @@
 eclipse.preferences.version=1
-# We target Java 1.7
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
-org.eclipse.jdt.core.compiler.compliance=1.7
-org.eclipse.jdt.core.compiler.source=1.7
-# Lines should be splitted at 140 chars
-org.eclipse.jdt.core.formatter.lineSplit=140
-# Indentation is 4 spaces
-org.eclipse.jdt.core.formatter.tabulation.char=space
-org.eclipse.jdt.core.formatter.tabulation.size=4
-# Configuration for NPE analysis
 org.eclipse.jdt.core.compiler.annotation.inheritNullAnnotations=enabled
 org.eclipse.jdt.core.compiler.annotation.missingNonNullByDefaultAnnotation=ignore
 org.eclipse.jdt.core.compiler.annotation.nullable=org.elasticsearch.common.Nullable
 org.eclipse.jdt.core.compiler.annotation.nullanalysis=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
+org.eclipse.jdt.core.compiler.compliance=1.7
+org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
```

Comment:
Sorry, I didn't notice that this had been added.

According to the documentation:  "When enabled, the compiler will issue an error or a warning when referring to a type with restricted access, as defined according to the access restriction specifications."

The default value is "warning".

I guess the version of Eclipse that originally generated these prefs didn't have that option, while mine is later and does have it.